# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fam/glibc.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Naurp fam-2.7.0-orig/src/Interest.h fam-2.7.0/src/Interest.h
--- fam-2.7.0-orig/src/Interest.h	2008-12-04 17:42:58.000000000 -0600
+++ fam-2.7.0/src/Interest.h	2008-12-04 17:45:12.000000000 -0600
@@ -29,6 +29,7 @@
 #include <netinet/in.h>  //  for in_addr
 
 #include "Boolean.h"
+#include <cstdlib>
 
 class Event;
 class FileSystem;
diff -Naurp fam-2.7.0-orig/include/BTree.h fam-2.7.0/include/BTree.h
--- fam-2.7.0-orig/include/BTree.h	2003-01-19 22:22:30.000000000 -0600
+++ fam-2.7.0/include/BTree.h	2008-12-04 17:47:04.000000000 -0600
@@ -24,6 +24,7 @@
 #define BTree_included
 
 #include "Boolean.h"
+#include <cstdlib>
 
 //  This is an in-core B-Tree implementation.
 //

